package-lock.json

```diff
@@ -1,5127 +1,8 @@
 {
   "name": "action-setup-enos",
-  "version": "0.0.12",
-  "lockfileVersion": 2,
+  "version": "0.0.14",
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Weird, this seems like a yuuuuge diff

Reply:
yeah...

Reply:
I did run a second npm install after seeing this comment and ended up with the same results. For posterity, this was done with npm version `6.14.17`.